pkg/config/field.go

```diff
@@ -191,6 +191,14 @@ type SetFieldConfig struct {
 	// f17elem = *f17iter.DBSecurityGroupName
 	// ```
 	From *string `json:"from,omitempty"`
+	// resources:
```

Comment:
Can you include a description of the `To` field above this example YAML please? Also, I recommend encapsulating  the YAML with: ` ```yaml``` ` like the above `From` docstring.